adc/ad7779/ad7779-driver.c

```diff
@@ -36,8 +36,14 @@
 #define COL_CYAN    "\033[1;36m"
 #define COL_NORMAL  "\033[0m"
 
-#define LOG_TAG "ad7779-drv: "
-#define log_debug(fmt, ...)     do { printf(LOG_TAG fmt "\n", ##__VA_ARGS__); } while (0)
+#define LOG_TAG "ad7779: "
+
+#if !NDEBUG
```

Comment:
use `#ifndef` or `#if !defined(NDEBUG)`. This code would fail with `-Wundef` compilation parameter.

I would switch the contents of if-else not to use "double negation" (right now it's "if not no debug")

Reply:
Done